### DISCUSSION TITLE
Microsoft has a form to report false positives in Windows Defender.

### DISCUSSION THREAD

**Discussion by Plaer1** [valinet/ExplorerPatcher#3677] [Ideas]
[Here's the form.](https://www.microsoft.com/en-us/wdsi/filesubmission/?persona=HomeUser)

I think _*everyone*_ who sees this should fill it out, honestly even with a corporation as ev!l as M$ I wouldn't assume malice. In this case, it seems like due to the nature of this software it's entirely possible it flagged automatically. There's (always) a good chance I'm wrong but if I'm not we could possibly get this fixed!

![2024-09-11_1726079835](https://github.com/user-attachments/assets/bed445e9-a10a-42e9-8503-37708a5ae33e)

**Comment by Amrsatrio** [comment]
Sorry to bring bad news to you but I think you will have no luck.

https://github.com/valinet/ExplorerPatcher/issues/2873#issuecomment-1964177592

**Comment by dariuszwit** [comment]
I have received it from Bitdefender:
The app D:\OneDrivePrivNew\OneDrive\Downloads\ep_setup.exe infected with Gen:Suspicious.Cloud.2.@B0@aaXsGSBT was moved to quarantine. It is recommended that you run a System Scan to make sure your system is clean

**Reply by pyrates999** [reply]
> I have received it from Bitdefender: The app D:\OneDrivePrivNew\OneDrive\Downloads\ep_setup.exe infected with Gen:Suspicious.Cloud.2.@b0@aaXsGSBT was moved to quarantine. It is recommended that you run a System Scan to make sure your system is clean

see this: https://github.com/valinet/ExplorerPatcher/issues/3228

**Comment by Witmarquzoti** [comment]
As much as it would be nice to have it removed from the false positives, it gets flagged because of its functionality.

**Reply by mcabel** [reply]
which part of the functionality exactly?

**Reply by pyrates999** [reply]
this part of all AV's: https://en.wikipedia.org/wiki/Heuristic_analysis

**Reply by Amrsatrio** [reply]
This drops a file in C:\Windows, named dxgi.dll. Real malware do that to achieve persistence.
